lxd/network/openvswitch/ovn.go

```diff
@@ -178,34 +178,71 @@ func NewOVN(s *state.State) (*OVN, error) {
 		return nil, errors.Wrapf(err, "Failed to get OVN northbound connection string")
 	}
 
+	sbConnection, err := NewOVS().OVNSouthboundDBRemoteAddress()
+	if err != nil {
+		return nil, errors.Wrapf(err, "Failed to get OVN southbound connection string")
```

Comment:
We don't use Wrapf error anymore though 😀

Reply:
I'll give that one a pass though, but yeah, we should put that file on the list of ones to convert :)